app/src/features/surveys/observations/observations-table/import-obsevations/ImportObservationsButton.tsx

```diff
@@ -42,6 +42,14 @@ export interface IImportObservationsButtonProps {
    * @memberof IImportObservationsButtonProps
    */
   onFinish?: () => void;
+  /**
+   * Options to pass to the process csv submission endpoint.
+   *
+   * @type {{
+   *     surveySamplePeriodId?: number;
+   *   }}
+   * @memberof IImportObservationsButtonProps
+   */
```

Comment:
nit: Remove

(This might just be remnants from my old branch, which did get cleaned up before it went to dev, but depending on when you pulled it in, it might have founds its way back here).

Reply:
processOptions should still exist in the sampling-site ImportObservationsButton component, correct?